rust/serving/src/app.rs

```diff
@@ -149,19 +151,16 @@ async fn graceful_shutdown(handle: Handle) {
     handle.graceful_shutdown(Some(Duration::from_secs(30)));
 }
 
-async fn create_js_context() -> crate::Result<Context> {
-    // Check for user and password in the Jetstream configuration
-    let js_config = &config().jetstream;
-
+async fn create_js_context(js_config: &JetStreamConfig) -> crate::Result<Context> {
     // Connect to Jetstream with user and password if they are set
-    let js_client = match (
-        env::var(ENV_NUMAFLOW_SERVING_JETSTREAM_USER),
-        env::var(ENV_NUMAFLOW_SERVING_JETSTREAM_PASSWORD),
-    ) {
-        (Ok(user), Ok(password)) => {
+    let js_client = match js_config.auth.as_ref() {
```

Comment:
this will go away eventually right? we will directly write to ISB via the pipeline framework.

Reply:
Yes, for now I'm trying to get it to a working state with minimal changes.